sonar-project.properties

```diff
@@ -1,5 +1,6 @@
 sonar.projectKey=monstar-lab-oss_nestjs-starter-rest-api
 sonar.organization=monstar-lab-oss
+sonar.exclusions=test/*, **/*.spec.ts
```

Comment:
```
test/**
```
@sivanpayyadakath perhaps this would be more correct. :thinking:

Reply:
@yashmurty 
I think this might be the correct usage
`test/**/* `
Will create a PR and see what happens 🙇